test/ymlr/encode_test.exs

```diff
@@ -79,8 +80,8 @@ defmodule Ymlr.EncodeTest do
     end
 
     test "quoted strings - escape seq forces double quotes (tab char)" do
-      assert_identity_and_output("a\tb", ~s("a\tb"))
-      assert_identity_and_output("!a\tb", ~s("!a\tb"))
+      assert_identity_and_output("a\tb", ~s(a\tb))
+      assert_identity_and_output("!a\tb", ~s('!a\tb'))
```

Comment:
@mruoss  the test says "forces double quotes" - but you removed the double quotes ... 

=> still wip?

Reply:
Yeah.... did not really look at the test name. => should be renamed.

Reply:
there. fixed.